src/components/DropdownMenu.tsx

```diff
@@ -0,0 +1,219 @@
+import React from 'react';
+import styled from 'styled-components';
+import { ButtonVariant, applyButtonVariantStyles } from '../theme/buttons';
+import { palette } from '../theme/palette';
+
+const StyledDropdownMenu = styled.div`
+  position: relative;
+`;
+
+const StyledDropdownMenuButton = styled.button<{ variant: ButtonVariant; width?: string }>`
+  ${(props) => applyButtonVariantStyles(props.variant)}
+
+  align-items: center;
+  border: 0;
+  border-radius: 5px;
```

Comment:
Tiny thing but I think we decided rems for everything?

Reply:
I believe I copied this from the Button component but I could set both to 0.5rem?

Reply:
Sounds good, I was probably not thinking about it back then :P

Reply:
Changed this to 0.5rem